t/t7524-serialized-status.sh

```diff
@@ -400,7 +400,7 @@ EOF
 
 '
 
-test_expect_success 'ensure deserialize -v does not crash' '
+test_expect_failure 'ensure deserialize -v does not crash' '
```

Comment:
I had to add this because this commit started failing at the very last line. Somehow we are no longer tracing the rejection.

Reply:
I'm not seeing the failure.  I've tried it on both Mac and Linux and the original test succeeds.
I'll keep digging.

Does anyone what this output from the test suite means?
There are 16 tests, 0 failed, but it prints this unhelpful "TODO passed: 13" line.

```
t7524-serialized-status.sh                       (Wstat: 0 Tests: 16 Failed: 0)
  TODO passed:   13
```

Reply:
I'm seeing that TODO message for both the Linux clang and the Linux gcc runs, but the former is green and the latter is red.

From the Linux clang build:

```
Test Summary Report
-------------------
t1092-sparse-checkout-compatibility.sh           (Wstat: 0 Tests: 18 Failed: 0)
  TODO passed:   12
t7524-serialized-status.sh                       (Wstat: 0 Tests: 16 Failed: 0)
  TODO passed:   13
Files=950, Tests=24653, 745 wallclock secs ( 7.01 usr  1.59 sys + 999.76 cusr 424.57 csys = 1432.93 CPU)
Result: PASS
```

From the Linux gcc build:
```
Test Summary Report
-------------------
t1092-sparse-checkout-compatibility.sh           (Wstat: 0 Tests: 18 Failed: 0)
  TODO passed:   12
t5799-gvfs-helper.sh                             (Wstat: 256 Tests: 29 Failed: 4)
  Failed tests:  9-12
  Non-zero exit status: 1
t7524-serialized-status.sh                       (Wstat: 0 Tests: 16 Failed: 0)
  TODO passed:   13
Files=950, Tests=24653, 703 wallclock secs ( 6.28 usr  1.48 sys + 952.58 cusr 380.85 csys = 1341.19 CPU)
Result: FAIL
```

(It looks like the t5799 is causing the build to fail.)

Reply:
right, I know about the `t1092` failure. I've got a commit ready for that. I'm hoping to instead figure out why our branch makes that test start succeeding, and then upstream that change. I'll do some digging.

Reply:
The status cache is only in microsoft/git, so t7524 won't be present in upstream.